gems/quilt_rails/lib/tasks/quilt_rails_tasks.rake

```diff
@@ -0,0 +1,45 @@
+namespace :quilt do
+  desc "Install @shopify/react-server with common configuration"
+  task install: :environment do
+    puts "Installing @shopify/react-server and @shopify/sewing-kit..."
+    system("yarn add @shopify/sewing-kit @shopify/react-server")
+
+    uiDir = "/app/ui"
+    fullUiPath = "#{Rails.root}#{uiDir}"
+
+    FileUtils.mkdir(fullUiPath) unless Dir.exists?(fullUiPath)
+
+    filesToWrite = {"#{uiDir}/index.tsx" =>
+"import * as React from 'react';
+import {AppProvider, Page, Card} from '@shopify/polaris';
+
+function App() {
+  return (
+    <AppProvider>
+      <Page title=\"Hello\">
+        <Card sectioned>Hi there</Card>
+      </Page>
+    </AppProvider>
+  );
+}
+
+export default App;
+    ",
+    "/config/routes.rb" => "mount Quilt::Engine, at: \"/\""}
```

Comment:
could we put these into files and read them in so we get syntax highlighting and stuff?

Reply:
In hindsight, these are useless and I am going to 🔥 them. We can't possibly add the route mounting bit without blowing away their existing stuff and I don't think I want to assume Polaris when adding a base app for them. Happy to revisit this in another PR.